### PR TITLE
clear logs on app launch

### LINKED ISSUE #1075: `flutter start` should automatically clear the logs before the app is run
To reduce the amount of clutter from old runs.

### PR DESCRIPTION
**Pull request by devoncarew**
Clear logs by default on app launch - fix https://github.com/flutter/flutter/issues/1075.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter_tools/lib/src/commands/listen.dart [hunk 1]
```diff
@@ -11,8 +11,8 @@ import 'start.dart';
 
 class ListenCommand extends StartCommandBase {
   final String name = 'listen';
-  final String description = 'Listen for changes to files and reload the running app on all connected devices. (Android only.)'
-      'By default, only listens to "./" and "./lib/". To listen to additional directories, list them on the command line.';
+  final String description = 'Listen for changes to files and reload the running app on all connected devices (Android only).'
+      ' By default, only listens to "./" and "./lib/". To listen to additional directories, list them on the command line.';
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/listen.dart]: please make sure logs, trace, listen, and start are all consistent in how this is punctuated.

**Comment by Hixie** [comment]
LGTM

**Comment by devoncarew** [comment]
Yup, looks like they're consistent.

**Pull request merged**